hathor/cli/run_node.py

```diff
@@ -232,10 +233,38 @@ def create_wallet():
         for description in args.listen:
             self.manager.add_listen_address(description)
 
-        self.start_manager()
+        self.start_manager(args)
         self.register_resources(args)
 
-    def start_manager(self) -> None:
+    def start_sentry_if_possible(self, args: Namespace) -> None:
+        """Start Sentry integration if possible."""
+        if not args.sentry:
+            return
+        self.log.info('Starting Sentry', dsn=args.sentry)
+        try:
+            import sentry_sdk
+        except ModuleNotFoundError:
+            self.log.error('ERROR: You must install the package \'sentry-sdk\' to enable sentry.')
+            sys.exit(-3)
+
+        import hathor
+        sentry_sdk.init(
```

Comment:
Can we have more information about the hostname/entrypoint here? It would be good to know which node generated the exception.

Reply:
Maybe this is the way: https://github.com/kiwicom/structlog-sentry#sentry-tags

Reply:
I can set the `server_name` to the peer-id. Otherwise, it will send the actual hostname. I feel that the actual hostname might be more useful if we configure it properly during deploy. What do you think?

Reply:
We can change the instance hostname during deploys, if needed.

We could do something like this to extract the Name of the instance and set it as the hostname inside it, without having to rely on passing input from aws-deploy-tools scripts: https://stackoverflow.com/a/24549602/3477266

But I think that it would be better to do this without having to set the hostname, just setting an ENV VAR maybe, and a custom tag in Sentry. I don't know if there are other bad consequences of changing the hostname.

Reply:
@luislhl I'll add a custom tag in Sentry in a separate PR. Let's see what other pieces of information are useful after we start using it for real.